public/app/features/alerting/unified/features.ts

```diff
@@ -13,10 +13,6 @@ const FEATURES: FeatureDescription[] = [
     name: AlertingFeature.NotificationPoliciesV2MatchingInstances,
     defaultValue: config.featureToggles.alertingNotificationsPoliciesMatchingInstances,
   },
-  {
-    name: AlertingFeature.ContactPointsV2,
```

Comment:
No longer needed?

Reply:
Nope, switching these to the standardized feature toggles now :)